### PR TITLE
Gradle changes

### PR DESCRIPTION
**Pull request by ziccardi**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.0.1'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.0'
-
+        classpath 'net.openid:appauth:0.7.0'
```
Comment (wei-lee) [inline build.gradle]: I don't think this should be at the top level build.gradle file. This is only required by the auth module, so I say move it to the `auth/build.gradle` file and it should be an implementation dependency.

File: auth/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.aerogear.android.core" />
```
Comment (danielpassos) [inline auth/src/main/AndroidManifest.xml]: s/package="org.aerogear.android.core"/package="org.aerogear.android.auth" ?

File: core/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,2 +1,2 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.aerogear.android.core" />
+    package="org.aerogear.auth" />
```
Comment (danielpassos) [inline core/src/main/AndroidManifest.xml]: Why core package was changed?

File: build.gradle (hunk 1 above)
Comment (secondsun) [reply build.gradle]: Just +1ing wei's comment.  Additionally this block is the dependencies for building (think devDependencies from node.js) as opposed to dependencies for the code.

**Comment by secondsun** [comment]
@ziccardi bump

**Comment by danielpassos** [comment]
@ziccardi Can you rebase and fix the conflict?

**Comment by secondsun** [comment]
So right now there is only one line in this file, is it safe to assume that these changes are part of another PR that has been merged?

**Comment by wtrocki** [comment]
Let's close this as it's surely outdated.

**Comment by ziccardi** [comment]
Yes, closing it now

**Pull request closed without merging**